code-challenges/string.md

```diff
@@ -107,8 +124,25 @@ front_times('Chocolate', 2) → 'ChoCho'
 front_times('Chocolate', 3) → 'ChoChoCho'
 front_times('Ok', 4) → 'OkOkOkOk'
 
-Needs sample solution
+Sample solution
+```
+function front_times(str, n) {
 
+  var front = str.substring(0, 3);
+  var result = "";
+  for (var i = 0; i < n; i++) {
```

Comment:
```suggestion
  const front = str.substring(0, 3);
  let result = "";
  for (let i = 0; i < n; i++) {
```

Please use `const` or `let` instead of `var`.

https://medium.com/@codingsam/awesome-javascript-no-more-var-working-title-999428999994

tl;dr:

- Use `const` when a variable won't be reassigned (i.e. you won't use an equal sign on it again)
- Use `let` when you need to reassign a value (i.e. you'll use an equal sign on it in the future)